releasenotes/notes/bogota-manila-rome-santiago-as-fakepulsebackends-2907dec149997a27.yaml

```diff
@@ -0,0 +1,6 @@
+---
+prelude: >
+    FakeBogota, FakeManila, FakeRome, and FakeSantiago can now be used as fake backends in Pulse experiments.
```

Comment:
You don't need a prelude section here. This section is reserved for the high level overview/highlights of the release. The contents of all the prelude sections are merged together into a single text section so we typically just write a signle release note with this section at release time.

```suggestion
```

Reply:
Ah ok. Thank you for clarifying that @mtreinish !